src/index.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable */
+import '@fortawesome/fontawesome-free/js/all.js';
+import './style.css';
+
+const cont = document.querySelector('.small-container');
+
+// addStoreList(mychore);
+
+const todoList = [{
+  index: '1',
+  list: 'wash the car',
+  completed: false,
+
+}, {
+  index: '2',
+  list: 'take out the trash',
+  completed: false,
+
+},
+{
+  index: '3',
+  list: 'get some javascript done',
+  completed: true,
+
+},
+{
+  index: '4',
+  list: 'do the dishes',
+  completed: false,
+
+}];
+
+for (let i = 0; i < todoList.length; i += 1) {
+  const div = document.createElement('div');
+
+  div.innerHTML = `
+            <div class ="checkclass">
+            <input class= "checklist" type="checkbox" id="${todoList[i].index}">
+            <label for="${todoList[i].index}">${todoList[i].list}</label>
+            </div>
+           
+           <i class="del-btn fa-solid fa-ellipsis-vertical">...</i> 
+            `;
+
+  div.classList.add('listItems');
+  cont.appendChild(div);
+}
```

Comment:
**[REQUIRED CHANGES]**

- Please, let's try to follow the project requirement by placing this block of code in a function to meet the project requirement below:

![func](https://user-images.githubusercontent.com/59397159/171673653-108937c5-bf76-4b67-a1ea-b69603a39a11.png)